app/models/manageiq/providers/ansible_tower/shared/inventory/persister/automation_manager.rb

```diff
@@ -1,21 +1,23 @@
 module ManageIQ::Providers::AnsibleTower::Shared::Inventory::Persister::AutomationManager
+  extend ActiveSupport::Concern
+  include ManagerRefresh::Inventory::AutomationManager
+
+  included do
+    has_automation_manager_credentials
```

Comment:
I have opted for dropping a `_inventory` suffix here.

So its either
`has_inventory { :model_class ...`
or
`has_<inventory name>`

Also note, that you can always overwrite any options